services/azure-devops/azure-devops-helpers.js

```diff
@@ -23,15 +23,8 @@ async function fetch(serviceInstance, { url, qs = {}, errorMessages }) {
   return { status }
 }
 
-function getHeaders() {
-  const headers = {}
-  if (serverSecrets.azure_devops_token) {
-    const pat = serverSecrets.azure_devops_token
-    const auth = Buffer.from(`:${pat}`).toString('base64')
```

Comment:
In my experience working with the AzDO APIs, this has always been a necessary step for using PATs vs. username/password. At first glance, I suspect that this change would actually break self-hosted Shields instances connecting to AzDO. 

When I get a chance I'll check out this branch and test it though to find out definitively

Reply:
It'd be great if you could try it.

I _think_ this code is manually constructing an HTTP Basic auth header with an empty username: https://stackoverflow.com/a/43843163/893113

That's ultimately [what request is doing, too](https://github.com/request/request#http-authentication).

Reply:
Yeah, and it _should_ work but for whatever reason I could never get things to work this way. Hopefully that's just a user error/fluke though (or something that's since been resolved) 🤞

Reply:
It works! This is one of those times that I am thrilled that I was wrong 😄